steps/steps-app-gallery-reports/0.9.0/step.yml

```diff
@@ -0,0 +1,245 @@
+title: Huawei AppGallery Connect Reports
+summary: Gets the different Reports from AppGallery Connect for a specific published
+  app.
+description: |-
+  Gets the next reports from AppGallery Connect:
+  - Download and Installation Report
+  - In-app Payment Report
+  - Paid Download Report
+  - Installation Failure Data Report
+
+  This step is based on [AppGallery Connect Reports](https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-References/agcapi-appdownloadexport)
+website: https://github.com/ferPrieto/steps-app-gallery-reports
+source_code_url: https://github.com/ferPrieto/steps-app-gallery-reports
+support_url: https://github.com/ferPrieto/steps-app-gallery-reports/issues
+published_at: 2021-03-06T00:13:45.025487Z
+source:
+  git: https://github.com/ferPrieto/steps-app-gallery-reports.git
+  commit: 6c74a96cdf30aa2ce82f753ae6f33a702e644916
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
```

Comment:
What about other multiplatform types?

Reply:
I don't think that's necessary to list, since this has been already submitted in the previous step this way

Reply:
Could you elaborate on that?

Reply:
This is just an optional parameter, it's enough by indicating the main platforms covered I already indicated. Nothing to elaborate here

Reply:
As per doc: https://devcenter.bitrise.io/contributors/create-your-own-step/#platform-types 
> select all platform types for which your Step is available

So you should list also `ionic` and `cordova`.

As elaborating I meant what was the motivation of not listing them when step was submitted.